samples/unity-of-bugs/Assets/Resources/Sentry/CustomOptionsConfiguration.asset

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
```

Comment:
IDK when this file disappeared - took me a bit of time to figure out why the CustomOptionsConfiguration wasn't showing up in the select window in the options...

Reply:
I think it got lost when I moved all the `SmokeTest` stuff out of the sample project.